pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -186,6 +187,14 @@ func computeReleasePayloadAcceptedCondition(payload *v1alpha1.ReleasePayload) me
 		Reason: ReleasePayloadAcceptedReason,
 	}
 
+	// If the release creation job failed, then the payload will never be Accepted
+	if payload.Status.ReleaseCreationJobResult.Status == v1alpha1.ReleaseCreationJobFailed {
```

Comment:
I wouldn't expect this to be necessary.  If you have no blocking jobs created, then you should be unknown or failed.

Reply:
This particular check is update the `PayloadAccepted` condition if/when `ReleaseCreationJobResult` fails to create the physical release.  If the release is not successfully created then it'll be impossible for this release to ever be `Accepted`.